meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://dl.viam.dev/module.schema.json",
```

Comment:
this was generated by the viam cli when i ran "viam create module" command which is needed before uploading the module. Honestly I only left it in because bill's meta.json had it still: [https://github.com/williamjhyland/viam-video-replay/blob/main/meta.json](https://github.com/williamjhyland/viam-video-replay/blob/main/meta.json)

Reply:
https://viaminc.slack.com/archives/C058VU5TM55/p1726086462872769

Reply:
Just going to keep it because it looks like it can be helpful to some vscode users